integration-test-common/src/test/java/com/datastrato/gravitino/common/integration/MiniGravitinoContext.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2023 Datastrato Pvt Ltd.
```

Comment:
We have just changed the package of this class, it's not a new class, so it's still 2023.